pkg/routers/health/buckets_test.go

```diff
@@ -0,0 +1,79 @@
+package health
+
+import (
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestTokenBucket_Take(t *testing.T) {
+	bucketSize := 10
```

Comment:
How do we decide on bucket size?

Reply:
Bucket size is essentially the number in the error budget config e.g. "10/s". So the bucket is how many errors we could tolerate during the give time frame.